mwdb/web/src/types/api.ts

```diff
@@ -317,3 +327,25 @@ export type OauthRemoveSingleProviderResponse = Response<null>;
 export type OauthGetLogoutLinkResponse = Response<{
     url: string;
 }>;
+
+export type UploadConfigRequest = {
```

Comment:
I would place these declarations after UploadFileRequest and Response for better readability